mem-01-b.xml

```diff
@@ -7525,7 +7647,9 @@ Ein besonderer Dank richtet sich an Marc Okrand ({marq 'oqranD:n:name}) für das
       <column name="examples_de"></column>
       <column name="examples_fa"></column>
       <column name="examples_sv"></column>
-      <column name="examples_ru"></column>
+      <column name="examples_ru">
+          ▶ {bortaS bIr jablu'DI' reH QaQqu' nay'.:sen} - Месть это блюдо, которое подают холодным
```

Comment:
Are you using an editor that auto-indents? Try to preserve the spacing/indentation, as it affects how the example is displayed in the app.

Also, this `examples_ru` column should've been left blank. The translations should be provided under the linked entries. The way this is done now, it will delete the {bortaS nIvqu' 'oH bortaS'e'} example from Russian, which I assume isn't what you want to do.

Reply:
I am using php-storm editor. I will check if it option enabled. As for `examples_ru` column. Will be the next version correct?
`
▶ {bortaS bIr jablu'DI' reH QaQqu' nay'.:sen} 
"Месть это блюдо, которое подают холодным"
`

Reply:
As I've mentioned, I've already merged your changes in https://github.com/De7vID/klingon-assistant-data/pull/93. You shouldn't have to fix anything.

See here for the current state of the {b} file at head:
https://github.com/De7vID/klingon-assistant-data/blob/master/mem-01-b.xml